packages/p2p/src/components/my-ads/my-ads-row-renderer.jsx

```diff
@@ -79,11 +84,36 @@ const MyAdsRowRenderer = observer(({ row: advert }) => {
     const handleOnEdit = () =>
         enable_action_point && floating_rate_store.rate_type !== rate_type ? onClickSwitchAd() : onClickEdit();
 
+    const should_show_tooltip_icon =
+        (visibility_status?.length === 1 && visibility_status?.[0] !== 'advert_inactive') ||
```

Comment:
do we need to check `visibility_status` here? we're giving an empty array by default